worlds/rabi_ribi/existing_randomizer/constraints_graph.txt

```diff
@@ -1661,7 +1661,7 @@ DARKNESS & (
 
 {
 "edge": "SKY_BRIDGE_REGEN_UP_LEDGE -> SKY_BRIDGE_SLIDE_AREA",
-"prereq": "SLIDING_POWDER & (CARROT_BOMB | (ITM_VHARD & EXPLOSIVES_ENEMY))",
+"prereq": "SLIDING_POWDER & DARKNESS & (CARROT_BOMB | (ITM_VHARD & EXPLOSIVES_ENEMY))",
```

Comment:
(nit) DARKNESS should be the leading condition here and `SKY_BRIDGE_SLIDE_AREA -> SKY_BRIDGE_DARK_AREA`